resources/lang/en/users.php

```diff
@@ -132,6 +132,7 @@
             'spam' => 'Spamming',
             'unwanted_content' => 'Linking inappropriate content',
             'nonsense' => 'Nonsense',
+            'multi_accounting' => 'Multi-Accounting',
```

Comment:
```suggestion
            'multi_accounting' => 'Using multiple accounts',
```
"multi-accounting" feels like a slang to me
not sure if important 🤔

Reply:
Yeah, open for suggestion for this one.

Reply:
fwiw, there are multiple mentions to "multi-accounting" on the wiki: https://osu.ppy.sh/wiki/en/Help_Centre/Account_Restrictions